example/traffic_mgmt/odp_traffic_mgmt.c

```diff
@@ -272,10 +289,14 @@ static uint32_t create_profile_set(profile_params_set_t *profile_params_set,
 
 	odp_tm_shaper_params_init(&shaper_params);
 	shaper                          = &profile_params_set->shaper_params;
-	shaper_params.commit_rate       = shaper->commit_rate  * shaper_scale;
-	shaper_params.peak_rate         = shaper->peak_rate    * shaper_scale;
-	shaper_params.commit_burst      = shaper->commit_burst * shaper_scale;
-	shaper_params.peak_burst        = shaper->peak_burst   * shaper_scale;
+	shaper_params.commit_rate       = clamp_rate(shaper->commit_rate *
```

Comment:
I wonder if it is good to silently change the values in this example program. Maybe some printout should be done if clamping happens.